pkg/storage/fs/fs_test.go

```diff
@@ -1,14 +1,45 @@
 package fs
 
 import (
-	"path/filepath"
+	"os"
+	"testing"
+
+	"github.com/gomods/athens/pkg/storage/compliance"
+	"github.com/spf13/afero"
+	"github.com/stretchr/testify/require"
 )
 
-func (d *FsTests) TestLocationFuncs() {
-	r := d.Require()
-	storage := d.storage.(*storageImpl)
-	moduleLoc := storage.moduleLocation(module)
-	r.Equal(filepath.Join(d.rootDir, module), moduleLoc)
-	versionedLoc := storage.versionLocation(module, version)
-	r.Equal(filepath.Join(d.rootDir, module, version), versionedLoc)
+func TestBackend(t *testing.T) {
+	fs := afero.NewOsFs()
```

Comment:
what do you think about making this and the below a memory FS, so that these tests don't touch the system at all?

Reply:
@arschles totes, but I'd like us to keep the Benchmarks to compare FS vs Memory. Benchmarks don't run during CI/CD as of now, so we should be ok.